### DISCUSSION TITLE
Make it Pip package

### DISCUSSION THREAD

**Discussion by dan65t** [sherlock-project/sherlock#1767] [Polls]
let's a have a poll, to see how many like this project to be a pip package ??

**Comment by krischal** [comment]
Why most OSINT tools doesn't provide package/libraries?

**Comment by Junubian** [comment]
Indeed

**Comment by ppfeister** [comment]
Sherlock is available on PyPI at https://pypi.org/project/sherlock-project/

To install Sherlock, the command `pip install sherlock-project` can be ran.
After installation, Sherlock can be ran with simply `sherlock`.

Further information can be found on the wiki or in the readme.
